raiden/api/rest.py

```diff
@@ -1219,6 +1221,37 @@ def _set_channel_reveal_timeout(
         result = self.channel_schema.dump(updated_channel_state)
         return api_response(result=result)
 
+    def _set_fee_schedule(
+        self,
+        registry_address: TokenNetworkRegistryAddress,
+        channel_state: NettingChannelState,
+        fee_schedule: FeeScheduleState,
+    ):
+        log.debug("Set fee schedule")
+        if channel.get_status(channel_state) != ChannelState.STATE_OPENED:
+            return api_error(
+                errors="Can't update the fee schedule of a closed channel",
+                status_code=HTTPStatus.CONFLICT,
+            )
+        try:
+            self.raiden_api.set_fee_schedule(
```

Comment:
I don't see any validation for the fee schedule values. Maybe @palango can advise on what rules that need to stand true before we dispatch the state change.

Reply:
The rules for these were sorted of punted, you are right. But for the flat/proportional attributes the validations are being done by marshmallow at the rest.py level. Perhaps they should also be added on the python.py side?